src/tcache.c

```diff
@@ -165,7 +165,7 @@ tcache_bin_flush_small(tsd_t *tsd, tcache_t *tcache, cache_bin_t *tbin,
 
 	bool merged_stats = false;
 	unsigned dalloc_count = 0;
-	VARIABLE_ARRAY(edata_t *, dalloc_slabs, nflush);
+	VARIABLE_ARRAY(edata_t *, dalloc_slabs, nflush + 1);
```

Comment:
Why `nflush + 1`?

Reply:
It's not really necessary but to avoid an UB warning. See https://github.com/jemalloc/jemalloc/issues/1767.